Trappist/src/Promact.Trappist.Web/typings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
same, what is the need of typings if you already have declare let jsPdf: any defined